Extension/browser/firefox/lib/prefs.js

```diff
@@ -36,6 +36,10 @@ var Prefs = exports.Prefs = {
 		var url = "filters/filter_" + filterId + ".txt";
 		return self.data.url(url);
 	},
+	getLocalMobileFilterPath: function (filterId) {
```

Comment:
I don't think that it's a right solution to embed mobile filters into the add-on.

Just work as if these filters were not embedded at all, load them from the backend.